.eslintrc.js

```diff
@@ -177,6 +177,7 @@ module.exports = {
       rules: {
         'n/no-process-exit': 'off',
         'n/no-missing-require': 'off', // `require` of `elm.json`.
+        'n/no-extraneous-require': ['error', {allowModules: ['fs-extra']}],
```

Comment:
This is for files generated by `new-package`, correct?

Reply:
Yeah. Not sure this issue only surfaces with these changes, but it should've been in #303.